content/rc/how-to/importing-data.md

```diff
@@ -5,61 +5,123 @@ weight: $weight
+
+        Where:
+
+        - `bucketname` - Name of the S3 bucket
+        - `path` - Path to the file, if necessary
+        - `filename` - Filename of the RDB file, including the .gz suffix if the file is compressed
+
+### RDB File from a Google Cloud Storage (GCS) Bucket
+
+Before you import the RDB file through the Redis Cloud Essentials console, you must share the file from the Google Cloud Platform (GCP) console.
+
+To share and import an RDB file that is stored in a GCS bucket:
+
+1. Go to the GCP console and click on your GCP project.
+1. Click on the menu to open it, and select **Storage** to open the Storage browser and view your buckets.
+1. Click on the bucket where the RDB file is stored.
+1. Edit the file permissions:
+    1. Click on the RDB file menu to open it, and click **Edit permissions**.
+    1. Click **Add item**.
+    1. Enter the user details and access:
+       - In the Entity field of the new item, select **User**.
+       - In the Name field of the new item, enter: `service@redislabs-prod-clusters.iam.gserviceaccount.com`
+       - In the Access field of the new item, select **Reader**.
+    1. Click **Save**.
+1. In the Redis Cloud Essentials management console, go to the database that you want to import into.
+1. Click **Import**.
+1. Enter the details for the RDB file:
+    - Source Type - Select **Google Cloud Storage**.
+    - RDB file path - Enter the URL for RDB file: `gs://bucketname/[path/]filename.rdb[.gz]`
+
+        Where:
+        - `bucketname` - Name of the GCS bucket
+        - `path` - Path to the file
+        - `filename` - Filename of the RDB file, including the .gz suffix if the file is compressed
+
+### RDB File from an Azure Blob Storage (ABS) Container
 
-`<protocol>://[username][:password]@[hostname][:port]/[path/]filename.rdb[.gz]`
+To import an RDB file that is stored in an ABS container:
 
-Where:
+1. In the Redis Cloud Essentials management console, go to the database that you want to import into.
+1. Click **Import**.
+1. Enter the details for the RDB file:
+    - Source Type - Select **Azure Blob Storage**.
+    - RDB file path - Enter the URL for RDB file: `abs://:storage_account_access_key@storage_account_name/[container/]filename.rdb[.gz]`
 
-- *protocol*: the server's protocol, can be either ftp, ftps, http or
-    https.
-- *username*: your username, if needed.
-- *password*: your password, if needed.
-- *hostname*: the hostname or IP address of the server.
-- *port*: the port number of the server, if needed.
-- *path*: the path to the file, if needed.
-- *filename*: the filename of the RDB file, optionally compressed and
-    with the *.gz* suffix.
+        Where:
+        - `storage_account_access_key` - Primary access key to the storage account
+        - `storage_account_name` - Name of the storage account
+        - `url` - URL of the storage account
+        - `container` - Name of the container, if necessary
```

Comment:
When is it necessary and for whom? Do we mean "if it exists" or "if you named the container"?